Admin/AbstractAdmin.php

```diff
@@ -1042,7 +1042,7 @@ public function getActiveSubclassCode()
     /**
      * {@inheritdoc}
      */
-    public function getBatchActions()
+    final public function getBatchActions()
```

Comment:
I found a way to throw deprecation notices.

I'll make a PR once GitHub will be ok to see my new branch...